### PR TITLE
Add iOS branch

### PR DESCRIPTION
**Pull request by bitPogo**
## Description

This enables the iOS branch.

## How is it being implemented?

This makes adjustments to support also iOS. It uses Base64#encode and Base64#encodeToString as example, which is taken from Kotlins starter guide and modified to increase its readability.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the changelog accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: util/src/iosMain/kotlin/care/data4life/sdk/util/Base64.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+package care.data4life.sdk.util
+
+actual object Base64 {
+    private const val BASE64_ALPHABET: String = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/"
```
Comment (wmontwe) [inline util/src/iosMain/kotlin/care/data4life/sdk/util/Base64.kt:20]: I wanted to comment on not using the same test data as Android/Java, but I see it's in the follow up pull-request, maybe both could have been one.

**Review by wmontwe — APPROVED** [review]
Just noticed that some part are missing, ideally that would be one pull-request

**Comment by wmontwe** [comment]
I won't merge this an look at the follow-up pull-request to not have TODOs in the main branch, violating always release state rule.

File: util/src/iosMain/kotlin/care/data4life/sdk/util/Base64.kt (hunk 1 above)
Comment (bitPogo) [reply util/src/iosMain/kotlin/care/data4life/sdk/util/Base64.kt:20]: Should I merge them?

**Pull request closed without merging**